src/reference/asciidoc/smb.adoc

```diff
@@ -0,0 +1,109 @@
+[[smb]]
+== SMB Adapter
```

Comment:
DITTO: `Adapters` or `Support`.

I even would say that `Support` is better since we provide not only a couple of channel adapters, but also an `SmbSession` and based on it an `SmbRemoteFileTemplate`.